mqtt/mqtt3-fuzz/Cargo.toml

```diff
@@ -0,0 +1,25 @@
+[package]
```

Comment:
Ah, if you're importing this too, then I have a couple of scripts I use locally to run `afl` that we should add here as a README or something. Remind me when I come in tomorrow.

Reply:
I'm going to add it as a separate PR